core/src/toga/window.py

```diff
@@ -272,6 +272,9 @@ def close(self) -> None:
         undefined, except for :attr:`closed` which can be used to check if the window
         was closed.
         """
+        if getattr(self, "content", None) is not None:
+            self.content.clear()
```

Comment:
Why is this needed? AFAICT, setting the window to None should be sufficient, and will propagate to all children. Clearing the children from content means the ownership hierarchy of `content` will be destroyed.

Reply:
Thanks, I hadn't noticed `_remove()` also propagates to the children as well. Also, does ownership hierarchy matter when we are clearing all the widgets?